.github/workflows/lint.yml

```diff
@@ -0,0 +1,23 @@
+name: "Lint cloudwatch-sample-application-ros1"
+on:
+  pull_request:
+
+jobs:
+  ament_lint:
+    runs-on: ubuntu-latest
+    container:
+      image: rostooling/setup-ros-docker:ubuntu-bionic-ros-eloquent-ros-base-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        linter: [copyright, flake8, pep257, xmllint]
+    steps:
+      - run: sudo chown -R rosbuild:rosbuild "$HOME" .
```

Comment:
We shouldn't need this anymore since https://github.com/ros-tooling/setup-ros-docker/issues/7 has been resolved.